package.json

```diff
@@ -90,8 +94,8 @@
     "morgan": "1.2.0",
     "ms": "0.7.1",
     "node-sass": "3.7.0",
+    "notifications-panel": "1.1.13",
```

Comment:
definitely in the wrong place, but this change should be removed and shrinkwrap rebuilt. it's 1.1.14 on master.